CHANGELOG.md

```diff
@@ -1,27 +1,26 @@
 # Change Log
 
-## 0.0.14 (Jully 2nd, 2020)
+## 0.1.0 (July 2nd, 2020)
 
-- enhancement - Using vulnerability data from Snyk.
-- enhancement - Includes latest updated security vulnerabilities of `maven`, `node` and `pypi` ecosystems, also includes Snyk curated Security Advisories/Vulnerability Unique to Snyk.
+- enhancement - Integration with [Snyk Intel Vulnerability DB](https://snyk.io/product/vulnerability-database/). It adds value with the latest, fastest and more number of vulnerability coverage derived from numerous sources. It also includes Snyk curated unique and pre-published security advisories that come with early detection and faster identification of vulnerabilities. Snyk Intel Vulnerability DB is the most advanced and accurate open source vulnerability database in the industry. Continuously curated by an experienced Security Research Team, the Snyk Intel Vulnerability DB maintains its high standards which enable your teams to be optimally efficient at containing open source security issues while maintaining their focus on development.
```

Comment:
It adds value with the latest, fastest and more number of vulnerability coverage derived from numerous sources

It adds value with the latest, fastest and more number of vulnerabilities derived from numerous sources